### PR TITLE
Create k8s

### PR DESCRIPTION
**Pull request by beckmani**

### CODE REVIEW

File: deployment/GCP/terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,78 @@
+provider "google" {
+  //  credentials = "${file("google_application_cred")}"
```
Comment (beckmani) [inline deployment/GCP/terraform/main.tf]: @zhernovskyi , can this be removed?

File: deployment/GCP/terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  project = "${var.project_name}"
+  zone    = "${var.zone}"
+}
+
+provider "google-beta" {
+  //  credentials = "${file("google_application_cred")}"
```
Comment (beckmani) [inline deployment/GCP/terraform/main.tf]: ditto

File: deployment/GCP/terraform/main.tf [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    use_ip_aliases = true
+  }
+
+  default_max_pods_per_node = 110
+
+  //  master_auth {
+  //    client_certificate_config {
+  //      issue_client_certificate = true
+  //    }
+  //  }
```
Comment (beckmani) [inline deployment/GCP/terraform/main.tf]: @zhernovskyi , can we remove this comment

File: deployment/GCP/terraform/outputs.tf [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    expiry: '{.credential.token_expiry}'
+    expiry-key: '{.credential.token_expiry}'
+    token-key: '{.credential.access_token}'
+current-context: ${google_container_cluster.cluster.name}-context
+contexts:
+- name: ${google_container_cluster.cluster.name}-context
+  context:
+    cluster: ${google_container_cluster.cluster.name}
+    user: ${google_container_cluster.cluster.name}-user    
+KUBECONFIG
```
Comment (beckmani) [inline deployment/GCP/terraform/outputs.tf]: @zhernovskyi , why we need this. We create the KUBECONFIG automatically by gcloud

**Review by beckmani — COMMENTED** [review]
See comments inside

File: deployment/GCP/terraform/main.tf (hunk 1 above)
Comment (zhernovskyi) [reply deployment/GCP/terraform/main.tf]: Sure, credentials will be used from env var GOOGLE_APPLICATION_CREDENTIALS

File: deployment/GCP/terraform/main.tf (hunk 2 above)
Comment (zhernovskyi) [reply deployment/GCP/terraform/main.tf]: Same

File: deployment/GCP/terraform/main.tf (hunk 3 above)
Comment (zhernovskyi) [reply deployment/GCP/terraform/main.tf]: Yes

File: deployment/GCP/terraform/outputs.tf (hunk 4 above)
Comment (zhernovskyi) [reply deployment/GCP/terraform/outputs.tf]: Can be removed. We don't use output values at all at the moment.

**Pull request merged**